src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$square-size: 100px;
+$day-of-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+.calendar {
+  width: 710px;
+  border: 1px solid black;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    background-color: #eee;
+    width: $square-size;
+    height: $square-size;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
```

Comment:
Let's specify the property for which you want to set up transition duration value